src/Context/ApiContext.php

```diff
@@ -169,8 +169,8 @@ public static function restore($fileName = self::FILENAME_CONFIG_DEFAULT)
     /**
      * @param $fileName
      *
-     * @return string[][]
      * @throws BunqException
+     * @return string[][]
```

Comment:
Our normal order is: first `@return` then `@throws`, motivated by the fact that the return is a normal and expected behaviour, whereas `@throws` is optional and exceptional. It makes sense to first highlight the main flow and then say "oh, and in case something goes wrong...".

@cafferata what's your point in changing this order?